src/pykiso/lib/auxiliaries/proxy_auxiliary.py

```diff
@@ -150,7 +154,7 @@ def get_proxy_con(
         for aux in aux_list:
             # aux_list can contain a auxiliary instance just grab the
             # channel
-            if isinstance(aux, AuxiliaryInterface):
+            if isinstance(aux, (AuxiliaryInterface, DTAuxiliaryInterface)):
                 self._check_compatibility(aux)
```

Comment:
It will raise the auxiliary creation error due to the fact that _dispatch_tx_method_to_channels is called.
But raise a error like WrongChannelTypeError could be a better way to inform the user

Reply:
I will add the log message when the double instance creation with start and create_instance is removed